tests/basic/pure_test.go

```diff
@@ -4238,3 +4238,115 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
 	})
 
 })
+
+var _ = Describe("{TestParallelPxAndFadaVolumeResize}", func() {
```

Comment:
Can we remove `Test` as a prefix for consistency? How about naming it `ValidateVolumeResizeInParallel` or something similar? It appears to resize FACD volumes, such as fio-cloudsnap, not just FADA or FBDA.